pages/user guides/financial_impact.md

```diff
@@ -0,0 +1,88 @@
+
+For all of these visuals, the majority of the data we're summarizing is sourced from the apps themselves, so that's where you'll want to look in order to add and tweak data. The only current exception to this is cloud instance targets, which are included in the cloud costs of your app. For more information about generating cloud instance targets, see our [guide](https://guides.tidal.cloud/instance-recommendation.html).
+
+{% include note.html content="For all apps, we also include the estimated migration cost in the projected cloud cost for the first year. You can see this visualized in time-series graphs such as' Current Vs Cloud Costs' and 'Savings Over Time'." %}
+
+## Current and Cloud Costs
+
+Arguably the most important data you'll be dealing with is the current cost and cloud cost for your apps. For the most part, 'cloud costs' refers to _rehost_ costs, however we do allow you to visualize and compare ROIs for alternative cloud migration transition types. More on that [later](#roi-comparison). 
+
+Current costs can be found in the 'Financial' category of the application overview. All currency fields which are in this category are counted towards the total annual current cost for this app.
+
+{% include image.html file="financial-impact-4.png" %}
+
+Cloud costs are found in the 'Transition Planning' category, which is under the 'Migration Plan' tab. The fields of note here are 'Forecasted Cost (Annually)' which is your estimated rehost cost, and 'Estimated Migration Cost', which is used as you might imagine. Note that this migration cost is critical for calculations such as return on investment and break-even point, and as noted earlier, this is included in your app's first year of cloud costs when visualized in time-series graphs.
+
+{% include image.html file="financial-impact-5.png" %}
+
+Do you have a cost that is not covered by these existing fields? Maybe you pay a license cost for this app, for example. That's easy to add, simply navigate to settings -> fields and create a new custom field by clicking the '+ New' button. The category you choose for this field determines whether it is a current or a cloud cost. For a current cost, select the 'Financial' category, and for a cloud cost, select the 'Transition Planning' category.
+
+{% include image.html file="financial-impact-6.png" %}
+
+After you've created your custom field, you can navigate back to the app record and populate it with data. Your financial impact visuals will be updated appropriately to reflect this new current or cloud cost.
+
+{% include image.html file="financial-impact-7.png" %}
+
+{% include note.html content="To be included in an app's current or cloud costs, the field must have category 'Financial' or 'Transition Planning' (for current and cloud costs respectively), and must have the field type 'Currency'. In most cases, you'll also want to set the field aggregation to 'Sum'." %}
+
+## Application Cost Breakdown
```

Comment:
Consider bringing this section before the creating new financial fields section.

Then remove references to custom fields, but instead explain how financial classification works. Then in the custom field section, you can more simply reference that the financial classification here is the same one talked about in application cost breakdown?